ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -203,20 +215,30 @@ def _create_model_path(model_path):
                 "permissions are set correctly.".format(model_path)
             )
 
+    @staticmethod
+    def _check_reset_params(reset_params, new_config):
```

Comment:
Just checking, where is this method called?

Reply:
Code trailing from use of lesson_controller; not needed anymore so removed from next update.